2018-09-29 04:21:30.467 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 04:21:30.470 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 04:22:30.467 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 04:22:30.470 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 04:23:17.720 20566 DEBUG oslo_concurrency.lockutils [req-ea85ec10-c5d6-4593-a022-c0b916a3508d 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:23:18.186 20566 DEBUG neutron.agent.dhcp.agent [req-085fcff9-6537-4421-a7db-0ecb82b37790 - - - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:23:18.187 20566 DEBUG neutron.agent.linux.utils [req-085fcff9-6537-4421-a7db-0ecb82b37790 - - - - -] Unable to access /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:18.188 20566 DEBUG neutron.agent.linux.dhcp [req-085fcff9-6537-4421-a7db-0ecb82b37790 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-b20afe9a-966c-43ae-9108-1866acba20b5 on network b20afe9a-966c-43ae-9108-1866acba20b5 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1272
2018-09-29 04:23:18.189 20566 DEBUG neutron.agent.linux.dhcp [req-085fcff9-6537-4421-a7db-0ecb82b37790 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-b20afe9a-966c-43ae-9108-1866acba20b5 on network b20afe9a-966c-43ae-9108-1866acba20b5 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1293
2018-09-29 04:23:21.149 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.167 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.181 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.201 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.214 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.231 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', 'link', 'set', 'tapb5403ebb-ec', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.262 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-o', 'link', 'show', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.278 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-29 04:23:21.300 20566 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tapb5403ebb-ec) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 04:23:21.302 20566 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-09-29 04:23:21.303 20566 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tapb5403ebb-ec) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 04:23:21.304 20566 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Port, col_values=(('tag', 4095),), record=tapb5403ebb-ec) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 04:23:21.306 20566 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=2): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'b5403ebb-ece9-4552-8914-bd7405ad8c98', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:11:67:f4'})), record=tapb5403ebb-ec) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 04:23:21.336 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapb5403ebb-ec', 'address', 'fa:16:3e:11:67:f4'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.349 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.360 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapb5403ebb-ec', 'netns', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.383 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', 'link', 'set', 'tapb5403ebb-ec', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.396 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', 'link', 'set', 'tapb5403ebb-ec', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.410 20566 DEBUG oslo_concurrency.lockutils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Acquired semaphore "iptables-qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:23:21.413 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.436 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.455 20566 DEBUG neutron.agent.linux.iptables_manager [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-09-29 04:23:21.457 20566 DEBUG oslo_concurrency.lockutils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Releasing semaphore "iptables-qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:23:21.458 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-o', 'link', 'show', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.469 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', 'addr', 'show', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.482 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'addr', 'add', '11.150.1.2/24', 'scope', 'global', 'dev', 'tapb5403ebb-ec', 'brd', '11.150.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.500 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.512 20566 DEBUG neutron.agent.linux.dhcp [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Setting IPv4 gateway for dhcp netns on net b20afe9a-966c-43ae-9108-1866acba20b5 to 11.150.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1169
2018-09-29 04:23:21.514 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.150.1.1', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.526 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.542 20566 DEBUG neutron.agent.linux.dhcp [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Cleaning stale devices for network b20afe9a-966c-43ae-9108-1866acba20b5 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1374
2018-09-29 04:23:21.543 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.555 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.597 20566 DEBUG neutron.agent.linux.dhcp [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:626
2018-09-29 04:23:21.599 20566 DEBUG neutron.agent.linux.dhcp [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/leases with contents:
1538295801 fa:16:3e:11:67:f4 11.150.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:654
2018-09-29 04:23:21.600 20566 DEBUG neutron.agent.linux.dhcp [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:23:21.601 20566 DEBUG neutron.agent.linux.dhcp [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:23:21.603 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Unable to access /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:21.605 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host', '--addn-hosts=/var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tapb5403ebb-ec', '--dhcp-range=set:tag0,11.150.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:21.635 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:21.640 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:21.642 20566 DEBUG neutron.agent.linux.external_process [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:23:21.643 20566 DEBUG oslo_concurrency.lockutils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:23:21.644 20566 DEBUG oslo_concurrency.lockutils [req-ea85ec10-c5d6-4593-a022-c0b916a3508d 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:23:22.198 20566 DEBUG neutron.agent.dhcp.agent [req-a927e876-8c13-415e-a4b4-d69fad74bab4 - - - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:23:22.199 20566 DEBUG neutron.agent.linux.dhcp [req-a927e876-8c13-415e-a4b4-d69fad74bab4 - - - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:23:22.206 20566 DEBUG neutron.agent.linux.dhcp [req-a927e876-8c13-415e-a4b4-d69fad74bab4 - - - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:23:22.212 20566 DEBUG neutron.agent.linux.utils [req-a927e876-8c13-415e-a4b4-d69fad74bab4 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:22.225 20566 DEBUG neutron.agent.linux.dhcp [req-a927e876-8c13-415e-a4b4-d69fad74bab4 - - - - -] Reloading allocations for network: b20afe9a-966c-43ae-9108-1866acba20b5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 04:23:22.227 20566 DEBUG neutron.agent.linux.utils [req-a927e876-8c13-415e-a4b4-d69fad74bab4 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:22.244 20566 DEBUG neutron.agent.linux.utils [req-a927e876-8c13-415e-a4b4-d69fad74bab4 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:22.257 20566 DEBUG neutron.agent.linux.utils [req-a927e876-8c13-415e-a4b4-d69fad74bab4 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:22.258 20566 DEBUG neutron.agent.linux.utils [req-a927e876-8c13-415e-a4b4-d69fad74bab4 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:22.259 20566 DEBUG neutron.agent.linux.external_process [req-a927e876-8c13-415e-a4b4-d69fad74bab4 - - - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:23:22.260 20566 DEBUG oslo_concurrency.lockutils [req-a927e876-8c13-415e-a4b4-d69fad74bab4 - - - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:23:22.261 20566 DEBUG oslo_concurrency.lockutils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:23:22.263 20566 DEBUG neutron.agent.dhcp.agent [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T08:23:18Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-b20afe9a-966c-43ae-9108-1866acba20b5, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'd1a72a3f-e0f0-4164-8a62-8625bcaa62c7', u'ip_address': u'11.150.1.2'}], id=b5403ebb-ece9-4552-8914-bd7405ad8c98, mac_address=fa:16:3e:11:67:f4, name=, network_id=b20afe9a-966c-43ae-9108-1866acba20b5, port_security_enabled=False, project_id=fe0a6898b40949349b729e4e791abbec, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=fe0a6898b40949349b729e4e791abbec, updated_at=2018-09-29T08:23:19Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:410
2018-09-29 04:23:22.264 20566 DEBUG oslo_concurrency.lockutils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:23:22.265 20566 DEBUG oslo_concurrency.lockutils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:23:22.266 20566 INFO neutron.agent.dhcp.agent [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T08:23:18Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-b20afe9a-966c-43ae-9108-1866acba20b5, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'd1a72a3f-e0f0-4164-8a62-8625bcaa62c7', u'ip_address': u'11.150.1.2'}], id=b5403ebb-ece9-4552-8914-bd7405ad8c98, mac_address=fa:16:3e:11:67:f4, name=, network_id=b20afe9a-966c-43ae-9108-1866acba20b5, port_security_enabled=False, project_id=fe0a6898b40949349b729e4e791abbec, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=fe0a6898b40949349b729e4e791abbec, updated_at=2018-09-29T08:23:20Z
2018-09-29 04:23:22.268 20566 DEBUG neutron.agent.dhcp.agent [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:23:22.269 20566 DEBUG neutron.agent.linux.dhcp [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:23:22.281 20566 DEBUG neutron.agent.linux.dhcp [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:23:22.286 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:22.304 20566 DEBUG neutron.agent.linux.dhcp [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Reloading allocations for network: b20afe9a-966c-43ae-9108-1866acba20b5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 04:23:22.305 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:22.319 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:22.331 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:22.332 20566 DEBUG neutron.agent.linux.utils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:22.333 20566 DEBUG neutron.agent.linux.external_process [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:23:22.334 20566 DEBUG oslo_concurrency.lockutils [req-1a155a1a-4209-44a2-b086-e8e2c45ee662 - - - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:23:30.467 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 04:23:30.471 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 04:23:53.449 20566 DEBUG oslo_concurrency.lockutils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:23:53.451 20566 INFO neutron.agent.dhcp.agent [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T08:23:52Z, description=, device_id=e9c1c676-be80-4b5a-a28f-ea79ddce43aa, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'd1a72a3f-e0f0-4164-8a62-8625bcaa62c7', u'ip_address': u'11.150.1.1'}], id=3c19ec6e-8acf-4f58-bbf9-ee56158e605b, mac_address=fa:16:3e:9f:9c:8b, name=, network_id=b20afe9a-966c-43ae-9108-1866acba20b5, port_security_enabled=False, project_id=fe0a6898b40949349b729e4e791abbec, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=fe0a6898b40949349b729e4e791abbec, updated_at=2018-09-29T08:23:52Z
2018-09-29 04:23:53.452 20566 DEBUG neutron.agent.dhcp.agent [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:23:53.454 20566 DEBUG neutron.agent.linux.dhcp [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:23:53.459 20566 DEBUG neutron.agent.linux.dhcp [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:23:53.464 20566 DEBUG neutron.agent.linux.utils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:53.484 20566 DEBUG neutron.agent.linux.dhcp [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Reloading allocations for network: b20afe9a-966c-43ae-9108-1866acba20b5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 04:23:53.485 20566 DEBUG neutron.agent.linux.utils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:53.501 20566 DEBUG neutron.agent.linux.utils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:53.514 20566 DEBUG neutron.agent.linux.utils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:53.515 20566 DEBUG neutron.agent.linux.utils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:53.516 20566 DEBUG neutron.agent.linux.external_process [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:23:53.517 20566 DEBUG oslo_concurrency.lockutils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:23:54.512 20566 DEBUG oslo_concurrency.lockutils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:23:54.513 20566 INFO neutron.agent.dhcp.agent [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T08:23:52Z, description=, device_id=e9c1c676-be80-4b5a-a28f-ea79ddce43aa, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'd1a72a3f-e0f0-4164-8a62-8625bcaa62c7', u'ip_address': u'11.150.1.1'}], id=3c19ec6e-8acf-4f58-bbf9-ee56158e605b, mac_address=fa:16:3e:9f:9c:8b, name=, network_id=b20afe9a-966c-43ae-9108-1866acba20b5, port_security_enabled=False, project_id=fe0a6898b40949349b729e4e791abbec, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=fe0a6898b40949349b729e4e791abbec, updated_at=2018-09-29T08:23:52Z
2018-09-29 04:23:54.514 20566 DEBUG neutron.agent.dhcp.agent [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:23:54.515 20566 DEBUG neutron.agent.linux.dhcp [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:23:54.523 20566 DEBUG neutron.agent.linux.dhcp [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:23:54.527 20566 DEBUG neutron.agent.linux.utils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:54.548 20566 DEBUG neutron.agent.linux.dhcp [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Reloading allocations for network: b20afe9a-966c-43ae-9108-1866acba20b5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 04:23:54.549 20566 DEBUG neutron.agent.linux.utils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:54.561 20566 DEBUG neutron.agent.linux.utils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:54.576 20566 DEBUG neutron.agent.linux.utils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:54.577 20566 DEBUG neutron.agent.linux.utils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:54.578 20566 DEBUG neutron.agent.linux.external_process [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:23:54.579 20566 DEBUG oslo_concurrency.lockutils [req-113d5485-ef50-4252-acb9-9246218711e0 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:23:57.254 20566 DEBUG oslo_concurrency.lockutils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:23:57.255 20566 INFO neutron.agent.dhcp.agent [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T08:23:52Z, description=, device_id=e9c1c676-be80-4b5a-a28f-ea79ddce43aa, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'd1a72a3f-e0f0-4164-8a62-8625bcaa62c7', u'ip_address': u'11.150.1.1'}], id=3c19ec6e-8acf-4f58-bbf9-ee56158e605b, mac_address=fa:16:3e:9f:9c:8b, name=, network_id=b20afe9a-966c-43ae-9108-1866acba20b5, port_security_enabled=False, project_id=fe0a6898b40949349b729e4e791abbec, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=fe0a6898b40949349b729e4e791abbec, updated_at=2018-09-29T08:23:53Z
2018-09-29 04:23:57.256 20566 DEBUG neutron.agent.dhcp.agent [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:23:57.258 20566 DEBUG neutron.agent.linux.dhcp [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:23:57.260 20566 DEBUG neutron.agent.linux.dhcp [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:23:57.264 20566 DEBUG neutron.agent.linux.utils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:57.282 20566 DEBUG neutron.agent.linux.dhcp [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Reloading allocations for network: b20afe9a-966c-43ae-9108-1866acba20b5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 04:23:57.283 20566 DEBUG neutron.agent.linux.utils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:57.302 20566 DEBUG neutron.agent.linux.utils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:57.317 20566 DEBUG neutron.agent.linux.utils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:57.318 20566 DEBUG neutron.agent.linux.utils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:57.319 20566 DEBUG neutron.agent.linux.external_process [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:23:57.319 20566 DEBUG oslo_concurrency.lockutils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:23:58.130 20566 DEBUG oslo_concurrency.lockutils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:23:58.132 20566 INFO neutron.agent.dhcp.agent [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T08:23:52Z, description=, device_id=e9c1c676-be80-4b5a-a28f-ea79ddce43aa, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'd1a72a3f-e0f0-4164-8a62-8625bcaa62c7', u'ip_address': u'11.150.1.1'}], id=3c19ec6e-8acf-4f58-bbf9-ee56158e605b, mac_address=fa:16:3e:9f:9c:8b, name=, network_id=b20afe9a-966c-43ae-9108-1866acba20b5, port_security_enabled=False, project_id=fe0a6898b40949349b729e4e791abbec, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=fe0a6898b40949349b729e4e791abbec, updated_at=2018-09-29T08:23:57Z
2018-09-29 04:23:58.134 20566 DEBUG neutron.agent.dhcp.agent [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:23:58.143 20566 DEBUG neutron.agent.linux.dhcp [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:23:58.145 20566 DEBUG neutron.agent.linux.dhcp [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:23:58.152 20566 DEBUG neutron.agent.linux.utils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:58.174 20566 DEBUG neutron.agent.linux.dhcp [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Reloading allocations for network: b20afe9a-966c-43ae-9108-1866acba20b5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 04:23:58.175 20566 DEBUG neutron.agent.linux.utils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:58.194 20566 DEBUG neutron.agent.linux.utils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:23:58.216 20566 DEBUG neutron.agent.linux.utils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:58.218 20566 DEBUG neutron.agent.linux.utils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:23:58.218 20566 DEBUG neutron.agent.linux.external_process [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:23:58.219 20566 DEBUG oslo_concurrency.lockutils [req-b932845d-8fc1-45d2-87fd-d18e5b7a0998 - - - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:24:11.341 20566 DEBUG oslo_concurrency.lockutils [req-36011176-f946-4399-a9cc-01024914e48f 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Acquired semaphore "dhcp-agent-network-lock-2050c613-9c50-45e4-b9c6-2203ec7994e3" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:24:11.342 20566 DEBUG oslo_concurrency.lockutils [req-36011176-f946-4399-a9cc-01024914e48f 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Releasing semaphore "dhcp-agent-network-lock-2050c613-9c50-45e4-b9c6-2203ec7994e3" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:24:15.727 20566 DEBUG oslo_concurrency.lockutils [req-70ab044d-1a0c-4f9f-ba01-bbbec5c419dc - - - - -] Acquired semaphore "dhcp-agent-network-lock-2050c613-9c50-45e4-b9c6-2203ec7994e3" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:24:15.728 20566 DEBUG oslo_concurrency.lockutils [req-70ab044d-1a0c-4f9f-ba01-bbbec5c419dc - - - - -] Releasing semaphore "dhcp-agent-network-lock-2050c613-9c50-45e4-b9c6-2203ec7994e3" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:24:16.751 20566 DEBUG oslo_concurrency.lockutils [req-70ab044d-1a0c-4f9f-ba01-bbbec5c419dc - - - - -] Acquired semaphore "dhcp-agent-network-lock-2050c613-9c50-45e4-b9c6-2203ec7994e3" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:24:16.752 20566 DEBUG oslo_concurrency.lockutils [req-70ab044d-1a0c-4f9f-ba01-bbbec5c419dc - - - - -] Releasing semaphore "dhcp-agent-network-lock-2050c613-9c50-45e4-b9c6-2203ec7994e3" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:24:30.467 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 04:24:30.471 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 04:24:35.953 20566 DEBUG oslo_concurrency.lockutils [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:24:35.955 20566 INFO neutron.agent.dhcp.agent [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T08:24:33Z, description=, device_id=a2213bba-9ce0-42cf-a24b-43abb1671aa6, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'd1a72a3f-e0f0-4164-8a62-8625bcaa62c7', u'ip_address': u'11.150.1.10'}], id=a31bfd6a-cb98-491a-ab37-9ebf40cfb11c, mac_address=fa:16:3e:3d:e5:1d, name=, network_id=b20afe9a-966c-43ae-9108-1866acba20b5, port_security_enabled=True, project_id=fe0a6898b40949349b729e4e791abbec, revision_number=3, security_groups=[u'c52904b5-321f-4e67-bb68-b0537d206d71'], status=DOWN, tags=[], tenant_id=fe0a6898b40949349b729e4e791abbec, updated_at=2018-09-29T08:24:34Z
2018-09-29 04:24:35.958 20566 DEBUG neutron.agent.dhcp.agent [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:24:35.960 20566 DEBUG neutron.agent.linux.dhcp [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:24:35.966 20566 DEBUG neutron.agent.linux.dhcp [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:24:35.971 20566 DEBUG neutron.agent.linux.utils [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:24:36.002 20566 DEBUG neutron.agent.linux.dhcp [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Reloading allocations for network: b20afe9a-966c-43ae-9108-1866acba20b5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 04:24:36.004 20566 DEBUG neutron.agent.linux.utils [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:24:36.023 20566 DEBUG neutron.agent.linux.utils [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:24:36.039 20566 DEBUG neutron.agent.linux.utils [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:24:36.040 20566 DEBUG neutron.agent.linux.utils [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:24:36.041 20566 DEBUG neutron.agent.linux.external_process [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:24:36.042 20566 DEBUG oslo_concurrency.lockutils [req-f0f87754-da38-4b53-b50b-da1be646ccb2 65fc2c9406b144e0959b78b722e98381 fe0a6898b40949349b729e4e791abbec - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:24:38.426 20566 DEBUG oslo_concurrency.lockutils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:24:38.427 20566 INFO neutron.agent.dhcp.agent [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T08:24:33Z, description=, device_id=a2213bba-9ce0-42cf-a24b-43abb1671aa6, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'd1a72a3f-e0f0-4164-8a62-8625bcaa62c7', u'ip_address': u'11.150.1.10'}], id=a31bfd6a-cb98-491a-ab37-9ebf40cfb11c, mac_address=fa:16:3e:3d:e5:1d, name=, network_id=b20afe9a-966c-43ae-9108-1866acba20b5, port_security_enabled=True, project_id=fe0a6898b40949349b729e4e791abbec, revision_number=4, security_groups=[u'c52904b5-321f-4e67-bb68-b0537d206d71'], status=DOWN, tags=[], tenant_id=fe0a6898b40949349b729e4e791abbec, updated_at=2018-09-29T08:24:36Z
2018-09-29 04:24:38.430 20566 DEBUG neutron.agent.dhcp.agent [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:24:38.437 20566 DEBUG neutron.agent.linux.dhcp [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:24:38.440 20566 DEBUG neutron.agent.linux.dhcp [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:24:38.449 20566 DEBUG neutron.agent.linux.utils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:24:38.470 20566 DEBUG neutron.agent.linux.dhcp [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Reloading allocations for network: b20afe9a-966c-43ae-9108-1866acba20b5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 04:24:38.476 20566 DEBUG neutron.agent.linux.utils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:24:38.509 20566 DEBUG neutron.agent.linux.utils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:24:38.531 20566 DEBUG neutron.agent.linux.utils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:24:38.532 20566 DEBUG neutron.agent.linux.utils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:24:38.533 20566 DEBUG neutron.agent.linux.external_process [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:24:38.534 20566 DEBUG oslo_concurrency.lockutils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:24:39.319 20566 DEBUG oslo_concurrency.lockutils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:24:39.320 20566 INFO neutron.agent.dhcp.agent [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T08:24:33Z, description=, device_id=a2213bba-9ce0-42cf-a24b-43abb1671aa6, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'd1a72a3f-e0f0-4164-8a62-8625bcaa62c7', u'ip_address': u'11.150.1.10'}], id=a31bfd6a-cb98-491a-ab37-9ebf40cfb11c, mac_address=fa:16:3e:3d:e5:1d, name=, network_id=b20afe9a-966c-43ae-9108-1866acba20b5, port_security_enabled=True, project_id=fe0a6898b40949349b729e4e791abbec, revision_number=5, security_groups=[u'c52904b5-321f-4e67-bb68-b0537d206d71'], status=DOWN, tags=[], tenant_id=fe0a6898b40949349b729e4e791abbec, updated_at=2018-09-29T08:24:38Z
2018-09-29 04:24:39.322 20566 DEBUG neutron.agent.dhcp.agent [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:24:39.323 20566 DEBUG neutron.agent.linux.dhcp [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:24:39.326 20566 DEBUG neutron.agent.linux.dhcp [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:24:39.331 20566 DEBUG neutron.agent.linux.utils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:24:39.349 20566 DEBUG neutron.agent.linux.dhcp [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Reloading allocations for network: b20afe9a-966c-43ae-9108-1866acba20b5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 04:24:39.352 20566 DEBUG neutron.agent.linux.utils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:24:39.367 20566 DEBUG neutron.agent.linux.utils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:24:39.382 20566 DEBUG neutron.agent.linux.utils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:24:39.383 20566 DEBUG neutron.agent.linux.utils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:24:39.383 20566 DEBUG neutron.agent.linux.external_process [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:24:39.384 20566 DEBUG oslo_concurrency.lockutils [req-bc91b743-cb15-47ba-8d29-1b812375e380 385405652b74463ca36119e213a84b2c e169fa6db0c147f283ae7c6a936117f3 - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:25:10.911 20566 DEBUG oslo_concurrency.lockutils [req-54eb4cb9-7abf-4138-a05b-b23ce6eb57c0 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Acquired semaphore "dhcp-agent-network-lock-2050c613-9c50-45e4-b9c6-2203ec7994e3" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:25:10.911 20566 DEBUG oslo_concurrency.lockutils [req-54eb4cb9-7abf-4138-a05b-b23ce6eb57c0 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Releasing semaphore "dhcp-agent-network-lock-2050c613-9c50-45e4-b9c6-2203ec7994e3" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:25:12.155 20566 DEBUG oslo_concurrency.lockutils [req-54eb4cb9-7abf-4138-a05b-b23ce6eb57c0 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Acquired semaphore "dhcp-agent-network-lock-2050c613-9c50-45e4-b9c6-2203ec7994e3" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:25:12.158 20566 DEBUG oslo_concurrency.lockutils [req-54eb4cb9-7abf-4138-a05b-b23ce6eb57c0 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Releasing semaphore "dhcp-agent-network-lock-2050c613-9c50-45e4-b9c6-2203ec7994e3" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:25:30.468 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 04:25:30.476 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.008s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 04:26:30.468 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 04:26:30.472 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 04:27:30.468 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 04:27:30.472 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 04:28:30.468 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 04:28:30.472 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 04:29:09.792 20566 DEBUG oslo_concurrency.lockutils [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:29:09.795 20566 DEBUG neutron.agent.dhcp.agent [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:29:09.796 20566 DEBUG neutron.agent.linux.utils [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'dhcp_release', 'tapb5403ebb-ec', '11.150.1.10', 'fa:16:3e:3d:e5:1d'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:29:09.849 20566 DEBUG neutron.agent.linux.dhcp [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:29:09.862 20566 DEBUG neutron.agent.linux.dhcp [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:29:09.879 20566 DEBUG neutron.agent.linux.utils [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:29:09.942 20566 DEBUG neutron.agent.linux.dhcp [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Reloading allocations for network: b20afe9a-966c-43ae-9108-1866acba20b5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 04:29:09.944 20566 DEBUG neutron.agent.linux.utils [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:29:09.967 20566 DEBUG neutron.agent.linux.utils [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:29:09.993 20566 DEBUG neutron.agent.linux.utils [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:29:09.995 20566 DEBUG neutron.agent.linux.utils [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:29:10.003 20566 DEBUG neutron.agent.linux.external_process [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:29:10.004 20566 DEBUG oslo_concurrency.lockutils [req-b4223768-1412-4b7c-af6b-f8d9f04495b4 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:29:30.469 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 04:29:30.473 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 04:30:04.131 20566 DEBUG oslo_concurrency.lockutils [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:30:04.134 20566 DEBUG neutron.agent.dhcp.agent [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:30:04.137 20566 DEBUG neutron.agent.linux.utils [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'dhcp_release', 'tapb5403ebb-ec', '11.150.1.1', 'fa:16:3e:9f:9c:8b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:30:04.180 20566 DEBUG neutron.agent.linux.dhcp [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Building host file: /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 04:30:04.195 20566 DEBUG neutron.agent.linux.dhcp [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Done building host file /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 04:30:04.202 20566 DEBUG neutron.agent.linux.utils [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:30:04.227 20566 DEBUG neutron.agent.linux.dhcp [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Reloading allocations for network: b20afe9a-966c-43ae-9108-1866acba20b5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 04:30:04.229 20566 DEBUG neutron.agent.linux.utils [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-4', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:30:04.257 20566 DEBUG neutron.agent.linux.utils [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5', 'ip', '-6', 'route', 'list', 'dev', 'tapb5403ebb-ec'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:30:04.273 20566 DEBUG neutron.agent.linux.utils [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:30:04.276 20566 DEBUG neutron.agent.linux.utils [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:30:04.277 20566 DEBUG neutron.agent.linux.external_process [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:30:04.281 20566 DEBUG oslo_concurrency.lockutils [req-90d0b8b0-5cff-4890-bbbc-0e2bde03aa6a 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:30:30.469 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 04:30:30.472 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 04:30:31.904 20566 DEBUG oslo_concurrency.lockutils [req-c0abd84d-1a88-4679-89d4-40b4aa1a2702 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:30:31.905 20566 DEBUG neutron.agent.dhcp.agent [req-c0abd84d-1a88-4679-89d4-40b4aa1a2702 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:30:31.907 20566 DEBUG neutron.agent.linux.utils [req-c0abd84d-1a88-4679-89d4-40b4aa1a2702 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Running command (rootwrap daemon): ['kill', '-9', '27878'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:30:31.929 20566 DEBUG neutron.agent.linux.utils [req-c0abd84d-1a88-4679-89d4-40b4aa1a2702 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-29 04:30:31.956 20566 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tapb5403ebb-ec) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 04:30:32.049 20566 DEBUG neutron.agent.linux.utils [req-3a52e900-6b45-4ce9-9089-a55bb9d41177 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:30:32.064 20566 DEBUG neutron.agent.linux.utils [req-3a52e900-6b45-4ce9-9089-a55bb9d41177 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:30:32.082 20566 DEBUG oslo_concurrency.lockutils [req-3a52e900-6b45-4ce9-9089-a55bb9d41177 - - - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:30:37.961 20566 DEBUG neutron.agent.dhcp.agent [req-a87c9722-583b-471e-9f59-c934f9ebc807 - - - - -] resync (b20afe9a-966c-43ae-9108-1866acba20b5): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:256
2018-09-29 04:30:37.962 20566 INFO neutron.agent.dhcp.agent [req-a87c9722-583b-471e-9f59-c934f9ebc807 - - - - -] Synchronizing state
2018-09-29 04:30:38.734 20566 INFO neutron.agent.dhcp.agent [req-fa410248-6bd2-4840-9536-ed9bb6441099 - - - - -] All active networks have been fetched through RPC.
2018-09-29 04:30:38.736 20566 INFO neutron.agent.dhcp.agent [-] Starting network 2050c613-9c50-45e4-b9c6-2203ec7994e3 dhcp configuration
2018-09-29 04:30:38.737 20566 INFO neutron.agent.dhcp.agent [-] Finished network 2050c613-9c50-45e4-b9c6-2203ec7994e3 dhcp configuration
2018-09-29 04:30:38.738 20566 INFO neutron.agent.dhcp.agent [-] Starting network b20afe9a-966c-43ae-9108-1866acba20b5 dhcp configuration
2018-09-29 04:30:38.738 20566 DEBUG neutron.agent.dhcp.agent [-] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:30:38.740 20566 DEBUG neutron.agent.linux.utils [-] Unable to access /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:30:38.740 20566 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-b20afe9a-966c-43ae-9108-1866acba20b5 on network b20afe9a-966c-43ae-9108-1866acba20b5 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1272
2018-09-29 04:30:38.741 20566 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-b20afe9a-966c-43ae-9108-1866acba20b5 on network b20afe9a-966c-43ae-9108-1866acba20b5 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1293
2018-09-29 04:30:41.887 20566 DEBUG neutron.agent.linux.dhcp [req-ac48a287-8590-42a0-a966-094875eb2579 - - - - -] Requested DHCP port with IPs on subnets set([u'd1a72a3f-e0f0-4164-8a62-8625bcaa62c7']) but only got IPs on subnets set([]). setup_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1346
2018-09-29 04:30:41.888 20566 DEBUG neutron.agent.linux.dhcp [req-ac48a287-8590-42a0-a966-094875eb2579 - - - - -] Cleaning stale devices for network b20afe9a-966c-43ae-9108-1866acba20b5 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1374
2018-09-29 04:30:41.889 20566 DEBUG neutron.agent.linux.utils [req-ac48a287-8590-42a0-a966-094875eb2579 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:30:41.909 20566 DEBUG neutron.agent.dhcp.agent [req-ac48a287-8590-42a0-a966-094875eb2579 - - - - -] Error configuring DHCP port, scheduling resync: Subnet on port 6f047ba0-588b-4e1e-bb85-19741bde4bbe does not match the requested subnet d1a72a3f-e0f0-4164-8a62-8625bcaa62c7. call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:153
2018-09-29 04:30:41.911 20566 INFO neutron.agent.dhcp.agent [req-ac48a287-8590-42a0-a966-094875eb2579 - - - - -] Finished network b20afe9a-966c-43ae-9108-1866acba20b5 dhcp configuration
2018-09-29 04:30:41.911 20566 INFO neutron.agent.dhcp.agent [req-fa410248-6bd2-4840-9536-ed9bb6441099 - - - - -] Synchronizing state complete
2018-09-29 04:30:41.913 20566 DEBUG oslo_concurrency.lockutils [req-f9fb9883-5a8d-41db-8102-12ba6ea58d71 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:30:43.012 20566 DEBUG neutron.agent.linux.utils [req-76c30073-6066-40a7-bd67-dfc8a1fab589 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:30:43.013 20566 DEBUG neutron.agent.linux.utils [req-76c30073-6066-40a7-bd67-dfc8a1fab589 - - - - -] Unable to access /var/lib/neutron/external/pids/b20afe9a-966c-43ae-9108-1866acba20b5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:30:43.014 20566 DEBUG neutron.agent.linux.external_process [req-76c30073-6066-40a7-bd67-dfc8a1fab589 - - - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:30:43.016 20566 DEBUG neutron.agent.dhcp.agent [req-76c30073-6066-40a7-bd67-dfc8a1fab589 - - - - -] Calling driver for network: b20afe9a-966c-43ae-9108-1866acba20b5 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 04:30:43.017 20566 DEBUG neutron.agent.linux.utils [req-76c30073-6066-40a7-bd67-dfc8a1fab589 - - - - -] Unable to access /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:30:43.018 20566 DEBUG neutron.agent.linux.utils [req-76c30073-6066-40a7-bd67-dfc8a1fab589 - - - - -] Unable to access /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 04:30:43.019 20566 DEBUG neutron.agent.linux.external_process [req-76c30073-6066-40a7-bd67-dfc8a1fab589 - - - - -] No process started for b20afe9a-966c-43ae-9108-1866acba20b5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 04:30:43.020 20566 DEBUG neutron.agent.linux.dhcp [req-76c30073-6066-40a7-bd67-dfc8a1fab589 - - - - -] Unable to access /var/lib/neutron/dhcp/b20afe9a-966c-43ae-9108-1866acba20b5/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:271
2018-09-29 04:30:43.020 20566 DEBUG neutron.agent.linux.dhcp [req-76c30073-6066-40a7-bd67-dfc8a1fab589 - - - - -] No interface exists for network b20afe9a-966c-43ae-9108-1866acba20b5 destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1487
2018-09-29 04:30:43.954 20566 DEBUG neutron.agent.linux.utils [req-fd83b2f1-4f79-47fb-ab22-96d98ff27558 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 04:30:43.968 20566 DEBUG neutron.agent.linux.dhcp [req-fd83b2f1-4f79-47fb-ab22-96d98ff27558 - - - - -] Namespace already deleted: qdhcp-b20afe9a-966c-43ae-9108-1866acba20b5 _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:249
2018-09-29 04:30:43.969 20566 DEBUG oslo_concurrency.lockutils [req-fd83b2f1-4f79-47fb-ab22-96d98ff27558 - - - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:30:43.970 20566 DEBUG oslo_concurrency.lockutils [req-ac48a287-8590-42a0-a966-094875eb2579 - - - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:30:43.971 20566 DEBUG neutron.agent.dhcp.agent [req-ac48a287-8590-42a0-a966-094875eb2579 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T08:30:39Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-b20afe9a-966c-43ae-9108-1866acba20b5, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'd1a72a3f-e0f0-4164-8a62-8625bcaa62c7', u'ip_address': u'11.150.1.2'}], id=6f047ba0-588b-4e1e-bb85-19741bde4bbe, mac_address=fa:16:3e:d8:e2:92, name=, network_id=b20afe9a-966c-43ae-9108-1866acba20b5, port_security_enabled=False, project_id=fe0a6898b40949349b729e4e791abbec, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=fe0a6898b40949349b729e4e791abbec, updated_at=2018-09-29T08:30:39Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:410
2018-09-29 04:30:43.972 20566 DEBUG oslo_concurrency.lockutils [req-ac48a287-8590-42a0-a966-094875eb2579 - - - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:30:43.973 20566 DEBUG oslo_concurrency.lockutils [req-ac48a287-8590-42a0-a966-094875eb2579 - - - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:30:43.973 20566 DEBUG neutron.agent.dhcp.agent [req-ac48a287-8590-42a0-a966-094875eb2579 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T08:30:39Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-b20afe9a-966c-43ae-9108-1866acba20b5, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[], id=6f047ba0-588b-4e1e-bb85-19741bde4bbe, mac_address=fa:16:3e:d8:e2:92, name=, network_id=b20afe9a-966c-43ae-9108-1866acba20b5, port_security_enabled=False, project_id=fe0a6898b40949349b729e4e791abbec, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=fe0a6898b40949349b729e4e791abbec, updated_at=2018-09-29T08:30:41Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:410
2018-09-29 04:30:43.974 20566 DEBUG oslo_concurrency.lockutils [req-ac48a287-8590-42a0-a966-094875eb2579 - - - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:30:45.460 20566 DEBUG oslo_concurrency.lockutils [req-f9fb9883-5a8d-41db-8102-12ba6ea58d71 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Acquired semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 04:30:45.462 20566 DEBUG oslo_concurrency.lockutils [req-f9fb9883-5a8d-41db-8102-12ba6ea58d71 81ed535ca88f461e8769becabf682408 67a7d8c970184497ad868e5ea6aa33b6 - - -] Releasing semaphore "dhcp-agent-network-lock-b20afe9a-966c-43ae-9108-1866acba20b5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 04:31:11.912 20566 DEBUG neutron.agent.dhcp.agent [req-fa410248-6bd2-4840-9536-ed9bb6441099 - - - - -] resync (b20afe9a-966c-43ae-9108-1866acba20b5): [SubnetMismatchForPort(u'Subnet on port 6f047ba0-588b-4e1e-bb85-19741bde4bbe does not match the requested subnet d1a72a3f-e0f0-4164-8a62-8625bcaa62c7.',)] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:256
2018-09-29 04:31:11.914 20566 INFO neutron.agent.dhcp.agent [req-fa410248-6bd2-4840-9536-ed9bb6441099 - - - - -] Synchronizing state
2018-09-29 04:31:12.760 20566 INFO neutron.agent.dhcp.agent [req-0a082f0e-c274-48ce-925f-86b3ed17032c - - - - -] All active networks have been fetched through RPC.
2018-09-29 04:31:12.762 20566 INFO neutron.agent.dhcp.agent [-] Starting network 2050c613-9c50-45e4-b9c6-2203ec7994e3 dhcp configuration
2018-09-29 04:31:12.763 20566 INFO neutron.agent.dhcp.agent [-] Finished network 2050c613-9c50-45e4-b9c6-2203ec7994e3 dhcp configuration
2018-09-29 04:31:12.763 20566 INFO neutron.agent.dhcp.agent [req-0a082f0e-c274-48ce-925f-86b3ed17032c - - - - -] Synchronizing state complete
2018-09-29 04:31:30.469 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 04:31:30.473 20566 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
